applications/st2110_demo/st2110_demo_app.py

```
@@ -0,0 +1,153 @@
+
+class NullSink(Operator):
+    """Sink operator that receives and discards frames.
+
+    Used to consume outputs that aren't connected to visualizers,
+    preventing scheduler deadlock from unconnected ports.
+    """
+
+    def setup(self, spec: OperatorSpec):
+        spec.input("input")
+
+    def compute(self, op_input, op_output, context):
```

Ruff (0.14.14)

[warning] Unused method argument: `op_output`

(ARG002)